manim/mobject/geometry.py

```diff
@@ -788,6 +900,37 @@ def __init__(self, vmob, alpha, length=1, d_alpha=1e-6, **kwargs):
 
 
 class Elbow(VMobject):
+    """Two lines that create a right angle about each other: L-shape.
+
+    Parameters
+    ----------
+    width : :class:`float`, optional
+        The length of the elbow's sides.
+    angle : :class:`float`, optional
+        The rotation of the elbow.
+    kwargs : Any
+        Additional arguments to be passed to :class:`~.VMobject`
+
+    Examples
+    --------
+
+    .. manim:: ElbowExample
+        :save_last_frame:
+
+        class ElbowExample(Scene):
+            def construct(self):
+                elbow_1 = Elbow()
+                elbow_2 = Elbow(width=4.0)
+                elbow_3 = Elbow(width=4.0, angle=5*PI/4)
```

Comment:
The values chosen here seem a bit large to me: <https://manimce--1128.org.readthedocs.build/en/1128/reference/manim.mobject.geometry.Elbow.html?highlight=elbow#manim.mobject.geometry.Elbow>

Reply:
My bad! I must have forgotten to update the docstring with my local changes. It should look as expected now.